core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -164,12 +210,33 @@ sealed private[data] trait KleisliFunctions {
   def lift[F[_], A, B](x: F[B]): Kleisli[F, A, B] =
     Kleisli(_ => x)
 
+  /**
+   * Creates a Kleisli ignoring input and lifting `A` into applicative context `F`.
+   * {{{
+   * scala> import cats.data.Kleisli, cats.implicits._
+   * scala> val pureOpt = Kleisli.pure[Option, Unit, String]("beam me up!")
+   * scala> pureOpt.run(())
+   * res0: Option[String] = Some(beam me up!)
+   * }}}
+   */
   def pure[F[_], A, B](x: B)(implicit F: Applicative[F]): Kleisli[F, A, B] =
     Kleisli(_ => F.pure(x))
 
+  /**
+   * Similar to [[pure]] except the input type is the same as the `A` in output type `F[A]`.
+   */
```

Comment:
This comment is true, but I think that it could be more illustrative. In my mind, `ask` is to `Kleisli` as `identity` is to `Function1`. I think that it would be helpful to point out that in the resulting `Kleisli`, the output `A` is the same as the input `A` (the actual value not just the type). A code example might help get this point across.

Reply:
Just updated and added an example to `ask`